HTMLhead.php

```diff
@@ -0,0 +1,5 @@
+<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
usually markup files like this we put into a folder called `includes` so they dont look like pages